apps/test/unit/lib/levelbuilder/rubrics/RubricsContainerTest.js

```diff
@@ -34,4 +116,48 @@ describe('RubricsContainerTest', () => {
     const afterAddLearningGoalItems = wrapper.find('LearningGoalItem').length;
     expect(afterAddLearningGoalItems).to.equal(initialLearningGoalItems + 1);
   });
+
+  it('changes the selected level for assessment when the dropdown is changed', () => {
+    const wrapper = shallow(<RubricsContainer {...defaultProps} />);
+    let dropdown = wrapper.find('select#rubric_level_id');
+    const dropdownValue = dropdown.prop('value');
+    expect(dropdownValue).to.equal(defaultProps.levels[0].id);
+
+    dropdown.simulate('change', {target: {value: defaultProps.levels[1].id}});
+    dropdown = wrapper.find('select#rubric_level_id');
+    expect(dropdown.prop('value')).to.equal(defaultProps.levels[1].id);
+  });
+
+  it('changes the saveNotificationText and disables the save Button when saving rubric', async () => {
+    const mockFetch = sinon.stub(global, 'fetch');
```

Comment:
this is a very creative solution! However we actually have a library which helps us with stubbing out network calls. please look for `sinon.fakeServer` and see if you can copy the pattern we use for that in other unit tests.